src/backend/matrix_client.py

```diff
@@ -1136,6 +1136,9 @@ async def room_join(self, alias_or_id_or_url: str) -> str:
         response = await super().join(string)
         return response.room_id
 
+    async def change_bookmark(self, room_id: str) -> None:
+        room = self.models[self.user_id, "rooms"].get(room_id)
+        room.bookmarked = not room.bookmarked
```

Comment:
```suggestion
    async def toggle_bookmark(self, room_id: str) -> None:
        room = self.models[self.user_id, "rooms"][room_id]
        room.bookmarked = not room.bookmarked
        
        bookmarks = self.backend.ui_settings["roomBookmarksIDs"].setdefault(self.user_id, [])
        if room.bookmarked and room_id not in bookmarks:
            bookmarks.append(room_id)
        elif not room.bookmarked and room_id in bookmarks:
            bookmarks.remove(room_id)
        
        await self.backend.ui_settings.write()
```
For the name: `toggle` makes what kind of change happens clearer

For the first line `room =`: It's not possible to right click and bookmark a room which isn't in the model, so no need to use `get` (besides, the line below would raise an error if `get` ever returned `None`)

With the code I added after the blank line, the config file will get edited accordingly when (un)bookmarking a room from the GUI.
Bookmarked room IDs should be stored per-account: a user might have multiple accounts which are part of a same room, but only want one of these account to have a bookmark for it.